### DISCUSSION TITLE
后端工程师怎么用这个？

### DISCUSSION THREAD

**Discussion by tiger2014** [klinecharts/KLineChart#285] [General]
我是做后端的，我有自己开发的交易策略测试系统，但是输出结果没有图表。看到这个库非常高兴，但是完全不知道怎么用，我对前端还停留在MVC的View的层面上。我不知道怎么在html上使用这个，能不能有一个Demo?

谢谢

**Comment by liihuu** [comment]
https://github.com/liihuu/KLineChartSample/tree/master/js-sample   这是一个简单的原生示例  可以看下
